packages/ui-focusable/src/Focusable/props.ts

```diff
@@ -37,16 +73,7 @@ type AllowedPropKeys = Readonly<Array<PropKeys>>
 type FocusableProps = FocusableOwnProps
 
 const propTypes: PropValidators<PropKeys> = {
-  /**
-   * @param {Object} renderProps
-   * @param {Boolean} renderProps.focused - Is the element focused (via keyboard only)?
-   * @param {HTMLElement} renderProps.focusable - The focusable element
-   * @param {Boolean} renderProps.focusVisible - Whether the focus state should be visible or not
```

Comment:
By removing this block this options don't show up on the docs page anymore, it only says `opts: FocusableRenderOptions` :/ It is not nice, but maybe we should add it back until find a way to render the docs better.

Reply:
I've added it back, but it looks wrong, I cant seem to specify an elements in an object properly (same issue in e.g. Editable/children). The original was even worse, it suggested that these functions have these as separate parameters when actually there is one object with these fields.